classes/QM_Collector_Memcache_Stats.php

```diff
@@ -1,16 +1,17 @@
 <?php
 
+declare( strict_types=1 );
```

Comment:
I agree we would want this, but there's no way the upstream would accept due to it locking out older versions of php, something wordpress are historically dead against. I think we shouldn't do this until we go all in on our fork (ideally they will merge your changes and we can just use the main one)

Reply:
Ah that makes sense. I'll use a different ruleset then (just added a random Automattic one to get started).

Reply:
@jenkoian changed to use WordPress-Extra and PHPCompatibilityWP :+1: